pontoon/contributors/templates/contributors/widgets/notifications_menu.html

```diff
@@ -1,14 +1,19 @@
 {% macro menu() %}
 {% if user.is_authenticated() %}
+
 {% set unread_notifications = user.notifications.unread() %}
-<div id="notifications" class="notifications select{% if unread_notifications|length > 0 %} unread{% endif %}">
+{% set unread_notifications_count = unread_notifications.count() %}
+{% set limit = 7 %}
```

Comment:
Sorry if that's a little bit to general question... but shouldn't we keep such numbers in pontoon-settings to avoid hardcoding? That's probably not the thing for this PR but I wonder in general how much pontoon should be extensible for installations outside Mozilla (if there are any).

Reply:
I agree, updated in the following commit.